### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by dahjelle** [felixkiss/atom-git-difftool#9]
I just noticed this error was popping up:

atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:54:11)
Object.activate (/Users/dahjelle/.atom/packages/git-difftool/lib/git-difftool.coffee:5:9)
```

**Comment by felixkiss** [comment]
Couldn't reproduce.

What version of Atom are you using? (I have the current `0.186.0`)

Where does the error pop up?

**Comment by felixkiss** [comment]
Fixed in `v0.2.4` (See ab28f3268e26ce1ee0bdbae158036079817a154b)
